kp_x_: 0.8
kp_y_: 0.8
kp_z_: 0.8

ki_x_: 0.000
ki_y_: 0.000
ki_z_: 0.000

kd_x_: 0.03
kd_y_: 0.03
kd_z_: 0.03

pose_x: 10.0
pose_y: 5.0
pose_z: 3.0